Library/Formula/zeromq.rb

```diff
@@ -1,8 +1,6 @@
 class Zeromq < Formula
   homepage "http://www.zeromq.org/"
-  url "http://download.zeromq.org/zeromq-4.0.5.tar.gz"
-  sha1 "a664ec63661a848ef46114029156a0a6006feecd"
-  revision 2
+  revision 3
```

Comment:
Agreed. But those who installed from source will, and it's possible they're just a highly vocal minority but we seem to have a decent chunk of source-builders around Brew. Happy to lose this discussion though, It's not something I love enough to fight for :smile_cat: